src/plugindata.h

```diff
@@ -59,7 +59,7 @@ G_BEGIN_DECLS
  * @warning You should not test for values below 200 as previously
  * @c GEANY_API_VERSION was defined as an enum value, not a macro.
  */
-#define GEANY_API_VERSION 235
+#define GEANY_API_VERSION 236
```

Comment:
This is the version currently in master as of bec3832359cf3a7d9f9d5d0a9ac3a622d6387d66, so changing it here is not very useful and likely to lead to conflicts or confusion.

Reply:
So I go up to 237? Or revert the change?